### ISSUE TITLE
"Error while loading rule" when using eslint's new flat config format

### ISSUE THREAD

**Issue by awwsmm** [buildo/eslint-plugin-no-loops#20]
MWE is something like

```js
// eslint.config.js

import noLoops from "eslint-plugin-no-loops";

export default [
  {
    plugins: {
      no_loops: noLoops
    },
    rules: {
      "no_loops/no-loops": 2
    }
  }
];
```

When running eslint with `eslint .`, this gives an error like

```
Oops! Something went wrong! :(

ESLint: 9.4.0

TypeError: Error while loading rule 'no_loops/no-loops': Rule must be an object with a `create` method
```

**Comment by awwsmm** [comment]
Until this is resolved, the compatibility layer provided by `eslint` seems to work as a patch

https://eslint.org/blog/2024/05/eslint-compatibility-utilities/#using-the-compatibility-utilities

```js
// eslint.config.js

import { fixupPluginRules } from "@eslint/compat";
import noLoops from "eslint-plugin-no-loops";

export default [
  {
    plugins: {
      no_loops: fixupPluginRules(noLoops)
    },
    rules: {
      "no_loops/no-loops": 2
    }
  }
];
```

**Comment by gabro** [comment]
Thanks for reporting, I've published version 0.4.0 for ESLint v9 compatibility (0.4.0 only supports v9 and up now)
